airbyte-webapp/src/pages/connections/ConnectionPage/ConnectionPageTitle.tsx

```diff
@@ -26,7 +26,8 @@ export const ConnectionPageTitle: React.FC = () => {
 
   const { connection } = useConnectionEditService();
 
-  const { data: freeConnectorProgramInfo } = useFreeConnectorProgram();
+  const { enrollmentStatusQuery } = useFreeConnectorProgram();
+  const { data: freeConnectorProgramInfo } = enrollmentStatusQuery;
   const displayEnrollmentCallout = freeConnectorProgramInfo?.showEnrollmentUi;
```

Comment:
can this be simplified to two lines instead of three by doing 

```suggestion
  const { enrollmentStatusQuery } = useFreeConnectorProgram();
  const displayEnrollmentCallout = enrollmentStatusQuery?.data?.freeConnectorProgramInfo?.showEnrollmentUi;
```
or something of the like? Just seems redundant as is with the intermediate steps there.